lib/config/error-codes/external.json

```diff
@@ -82,7 +82,8 @@
         "index_reserved": {
           "code": 17,
           "message": "Indexes starting with a '%' are reserved for internal use. Cannot process index %s.",
-          "class": "BadRequestError"
+          "class": "BadRequestError",
+          "deprecated": true
```

Comment:
Should I remove the error code instead ?

Reply:
Frankly, I don't know what to do with those.

Reply:
We should keep them, just to be sure we aren't gonna use the same code for a different error.  
And with Kuzzle 3.x we can get ride of them